monitor/plugins/spark_sahara/plugin.py

```diff
@@ -74,9 +72,9 @@ def _publish_measurement(self, job_request):
         jobs.reverse()
 
         if not len(jobs) == 0:
-            if self.current_job_id is 0:
-                self.first_submission_time = jobs[self.current_job_id]\
-                                                 ['submissionTime']
+            if self.current_job_id == 0:
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR?